Sources/Rendering/OpenGL/Renderer/index.js

```diff
@@ -136,7 +136,8 @@ function vtkOpenGLRenderer(publicAPI, model) {
 
     if (!model.renderable.getTransparent()) {
       const background = model.renderable.getBackground();
-      model.context.clearColor(background[0], background[1], background[2], 1.0);
+      if (background.length === 3) background.push(1.0);
```

Comment:
You could do the check in the vtkOpenGLRenderer(publicAPI, model) function. Only once at instantiation time.